pkg/filesystem/simple.go

```diff
@@ -275,5 +276,13 @@ func (f *SimpleFileSystem) PreDeactivate(ctx context.Context) (bool, error) {
 		return false, fmt.Errorf("could not unmount after pre-deactivate '%s': %w", f.TempDir, err)
 	}
 
-	return false, nil
+	return true, nil
+}
+
+func (f *SimpleFileSystem) PostMount(ctx context.Context, complete bool) (bool, error) {
+
+	// TODO: Make this the same as PostActivate? Rename PostActivate Here? Have them both do the same thing?
```

Comment:
Yeah, just rename PostActivate to PostMount. PostActivate will only be a Lustre thing. Or I guess PostActivate could be for XFS and GFS2 as well. It would just allow you to run commands after the mkfs but before a mount of the file system. I don't think that's useful though.

Reply:
I'll make PostActivate/Deactive for xfs/gfs2 a no-op.